src/app-layer-htp-range.c

```diff
@@ -139,7 +139,7 @@ static inline bool ContainerValueRangeTimeout(HttpRangeContainerFile *cu, const
     return false;
 }
 
-static void ContainerUrlRangeUpdate(HttpRangeContainerFile *cu, uint32_t expire)
+static void ContainerUrlRangeUpdate(HttpRangeContainerFile *cu, uint64_t expire)
```

Comment:
if this is expire time in seconds, u32 should be enough. If it is isn't we should convert to `SCTime`

Reply:
@jlucovsky what do you think about this ?
This comes from `SCTIME_SECS` returning a u64